code/game/machinery/portable_turret.dm

```diff
@@ -999,6 +999,7 @@ GLOBAL_LIST_EMPTY(turret_icons)
 
 	faction = "syndicate"
 	emp_vulnerable = FALSE
+	density = TRUE
```

Comment:
Syndie turrets not being dense is intentional.

I gamed too hard circa 2016-17 on the old depot.

Reply:
/s I would rather people have to use their brains and game the depot than people just grug stand on the turrets. but regardless its up to you, if you don't care for the change feel free to close the PR.

Reply:
I myself am not in favour, but thats up to others to decide